src/platform-includes/configuration/integrations/pluggable-integrations/javascript.mdx

```diff
@@ -0,0 +1,85 @@
+These pluggable integrations are snippets of code that augment functionality for specific applications and/or frameworks. We document them so you can see what they do and that they can be enabled.
+
+## How to Enable
+
+Install the `@sentry/integrations` package and provide a new instance with your config to `integrations` option. Include the plugin after the SDK has been loaded.
+
+For example:
+
+<PlatformContent includePath="configuration/enable-pluggable-integrations" />
+
+### ExtraErrorData
+
+_Import name: `Sentry.Integrations.ExtraErrorData`_
+
+This integration extracts all non-native attributes from the Error object and attaches them to the event as the `extra` data. If the error object has a `.toJSON()` method, the `ExtraErrorData` integration will run it to extract additional information.
+
+Available options:
+
+<PlatformContent includePath="configuration/extra-error-data" />
+
+### CaptureConsole
+
+_Import name: `Sentry.Integrations.CaptureConsole`_
+
+This integration captures all `Console API` calls and redirects them to Sentry using the SDK's `captureMessage` or `captureException` call depending on the log level. It then retriggers to preserve default native behavior.
+
+<PlatformContent includePath="configuration/capture-console" />
+
+### Debug
+
+_Import name: `Sentry.Integrations.Debug`_
+
+This integration allows you to inspect the content of the processed event, that will be passed to `beforeSend` and effectively send to the Sentry SDK. It will *always* run as the last integration, no matter when it was registered.
```

Comment:
Based on the Node version, I think this edit is correct
```suggestion
This integration allows you to inspect the content of the processed event that will be passed to `beforeSend`, and effectively send it to the Sentry SDK. It will *always* run as the last integration, no matter when it was registered.
```